File: pmcjs/scripts/sqloop.pl

use DBI;
for (my $i = 0; $i < 10000; $i++) {
  my $dbh = DBI->connect("dbi:SQLite:dbname=test.sq3","","");
  $dbh->prepare("SELECT 3")->execute();
  $dbh->disconnect();
}

File: pmcjs/scripts/sqloop.js

use('Perl');

Perl.Use("DBI");
for (var i = 0; i < 10000; ++i) {
  var dbh = Perl.Methods["DBI"].connect("dbi:SQLite:dbname=test.sq3","","");
  dbh["do"]("SELECT 3");
  dbh.disconnect();
  dbh.Detach();
  dbh = null;
}

